src/tools/data.js

```diff
@@ -14,6 +14,18 @@ const isPlainObj = o => {
 
 const comparison = (obj, needle) => obj === needle;
 
+const getObjectType = obj => {
+  if (_.isPlainObject(obj)) {
```

Comment:
Any reason we have our own `isPlainObject`? Is it preferred to lodash's?

Reply:
🤷‍♂️ but yeah, I think we should stick to lodash's.

Reply:
🤷‍♂️ but yeah, I think we should stick to lodash's.